go/apps/mixnet/mixnet_test.go

```diff
@@ -30,13 +32,24 @@ import (
 
 var password = make([]byte, 32)
 var network = "tcp"
-var routerAddr = "localhost:7007"
-var dstAddr = "localhost:7009"
+var proxyAddr = "127.0.0.1:1080"
+var routerAddr = "127.0.0.1:7007"
+var dstAddr = "127.0.0.1:7009"
+var timeout, _ = time.ParseDuration("1s")
```

Comment:
All of the tests (except obviously the timeout tests) finish well under 1 second; we could probably safely reduce this to 0.5 seconds without making the tests flaky.